### ISSUE TITLE
[Bug]: binutils man pages corrupted

### ISSUE THREAD

**Issue by Alto1772** [termux/termux-packages#7987]
### Problem description

Binutils package man pages contain nothing but ~~in some unknown binary context~~ a gzipped file with nothing on it.

![Screenshot_20211116_201930](https://user-images.githubusercontent.com/56553686/141984650-51093da6-ad56-41f1-bd2b-d6ed9771b76d.jpg)

### What steps will reproduce the bug?

`man ar`

### What is the expected behavior?

It should open a man page of a command

### System information

termux-info:

```
Application version:
0.117
Packages CPU architecture:
aarch64
Subscribed repositories:
# sources.list
deb https://deb.kcubeterm.me/termux-main/ stable main
# game-repo (sources.list.d/game.list)
deb https://packages.termux.org/apt/termux-games games stable
# science-repo (sources.list.d/science.list)
deb https://packages.termux.org/apt/termux-science science stable
Updatable packages:
All packages up to date
Android version:
10
Kernel build information:
Linux localhost 4.14.116 #1 SMP PREEMPT Thu Sep 23 21:21:02 CST 2021 aarch64 Android
Device manufacturer:
HUAWEI
Device model:
JNY-LX1

```

**Comment by thunder-coding** [comment]
Seems like the same problem as https://github.com/freebsd/freebsd-ports/commit/a055899f4f1c41d07ae5b8d3f5a9983173ef76f6#diff-fb241693cd37498fc6cded795a19b5b8793b8b25670ba02a7fafb4017b6ccbc6.

**Comment by Alto1772** [comment]
> Seems like the same problem as [freebsd/freebsd-ports@a055899#diff-fb241693cd37498fc6cded795a19b5b8793b8b25670ba02a7fafb4017b6ccbc6](https://github.com/freebsd/freebsd-ports/commit/a055899f4f1c41d07ae5b8d3f5a9983173ef76f6#diff-fb241693cd37498fc6cded795a19b5b8793b8b25670ba02a7fafb4017b6ccbc6).

thanks for the patch, that is also my problem in my cross-compiled toolchain built inside termux

**Comment by thunder-coding** [comment]
This should be fixed in 2.37-4, should be available from official repos in 5-10 minutes and upto 6 hours on mirrors
